### PR TITLE
DevWs for SHINA SYSTEM containing containing ZigBee Lock Without Codes

### PR DESCRIPTION
**Pull request by shinasys**

### CODE REVIEW

File: devicetypes/smartthings/zigbee-lock-without-codes.src/zigbee-lock-without-codes.groovy [hunk 1]
```diff
@@ -138,7 +142,7 @@ def initialize() {
 		cmds += zigbee.configureReporting(CLUSTER_DOORLOCK, DOORLOCK_ATTR_LOCKSTATE,DataType.ENUM8, 0, 3600, null) 
 		cmds += zigbee.configureReporting(CLUSTER_POWER, POWER_ATTR_BATTERY_PERCENTAGE_REMAINING,DataType.UINT8, 600, 21600, 0x01)
 		cmds += zigbee.readAttribute(CLUSTER_POWER, POWER_ATTR_BATTERY_PERCENTAGE_REMAINING)
-
+		if(isSiHASLock()) cmds += zigbee.configureReporting(CLUSTER_DOORLOCK, DOORLOCK_ATTR_DOORSTATE,DataType.ENUM8, 0, 3600, null)
```
Comment (PKacprowiczS) [inline devicetypes/smartthings/zigbee-lock-without-codes.src/zigbee-lock-without-codes.groovy]: nit: space after `if`

File: devicetypes/smartthings/zigbee-lock-without-codes.src/zigbee-lock-without-codes.groovy [hunk 2]
```diff
@@ -115,6 +117,8 @@ def refresh() {
 		cmds += zigbee.readAttribute(CLUSTER_IAS_ZONE, IAS_ATTR_ZONE_STATUS)
 	}
 
+	if(isSiHASLock()) cmds += zigbee.readAttribute(CLUSTER_DOORLOCK, DOORLOCK_ATTR_DOORSTATE)
```
Comment (PKacprowiczS) [inline devicetypes/smartthings/zigbee-lock-without-codes.src/zigbee-lock-without-codes.groovy]: nit: space after `if`

**Review by PKacprowiczS — CHANGES_REQUESTED** [review]

**Comment by PKacprowiczS** [comment]
@greens @dkirker this request adds `Contact Sensor` capability to the DTH, but existing setups shouldn't be affected, as vids in use don't support it

**Review by greens — APPROVED** [review]

**Comment by greens** [comment]
@PKacprowiczS lgtm, merge once you're happy with it

**Review by dkirker — APPROVED** [review]

**Review by PKacprowiczS — APPROVED** [review]

**Pull request merged**